src/components/pacbio/PacbioWellModal.vue

```diff
@@ -136,8 +149,19 @@ export default {
       this.mutateWell({ position: this.position, property: 'movie_time', with: movieTime })
     },
     updateSequencingMode(seqMode) {
+      this.preExtensionTimeValue(seqMode)
       this.mutateWell({ position: this.position, property: 'sequencing_mode', with: seqMode })
     },
+    updatePreExtensionTime(preExtensionTime) {
+      this.mutateWell({ position: this.position, property: 'pre_extension_time', with: preExtensionTime })
+    },
+    preExtensionTimeValue(seqMode) {
```

Comment:
I would call this method 'defaultPreExtensionTime', so it starts with a verb and is a bit more descriptive of what it does